README.md

```diff
@@ -35,7 +40,18 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+var instructor = Instructor.find({name: "Andy"}, function (err, docs){
+  if(err)
+console.log("Action cannot be completed")});
+console.log(docs);
+});
+var instructor = Instructor.create({gender: "Male"}, function (err, docs){
+  if(err)
+  console.log("Action cannot be completed")});
+  console.log(docs);
+});
+})
+
```

Comment:
The goal is not to create a new instructor. The goal is to create a new wishlist item that belongs to an instructor.